src/config/serverConfig.ts

```diff
@@ -1,14 +1,15 @@
 // SPDX-FileCopyrightText: 2024 PNED G.I.E.
 //
 // SPDX-License-Identifier: Apache-2.0
+import { env } from 'next-runtime-env';
 
 interface ServerConfig {
   ckanUrl: string;
   daamUrl: string;
 }
 
 const serverConfig: ServerConfig = {
-  ckanUrl: process.env.NEXT_PUBLIC_CKAN_URL || 'https://ckan-test.healthdata.nl',
+  ckanUrl: env('NEXT_PUBLIC_CKAN_URL') || 'https://ckan-test.healthdata.nl',
   daamUrl: process.env.DAAM_URL || 'http://localhost:8080',
```

Comment:
important: can you please also do it for `DAAM_URL`?

Reply:
check